packages/importer/src/importer.ts

```diff
@@ -174,24 +171,6 @@ export class OneSchemaImporterClass extends EventEmitter {
         sessionToken: mergedParams.sessionToken,
         ...baseMessage,
       }
-    } else if (mergedParams.templateGroupKey) {
-      message = {
-        messageType: "init-template-group",
-        userJwt: mergedParams.userJwt,
-        templateGroupKey: mergedParams.templateGroupKey,
-        importConfig: mergedParams.importConfig,
-        customizationKey: mergedParams.customizationKey,
-        customizationOverrides: mergedParams.customizationOverrides,
-        ...baseMessage,
-      }
-      if (!message.userJwt) {
```

Comment:
I think we need to keep this userJwt thing in

Reply:
Hum... but this is inside the `mergedParams.templateGroupKey` block, so would never be called, right?

Reply:
wait myb im blind ur right sorry